web/packages/teleport/src/UnifiedResources/Resources.tsx

```diff
@@ -38,12 +38,15 @@ import { useInfiniteScroll } from 'teleport/components/hooks/useInfiniteScroll';
 import { SearchResource } from 'teleport/Discover/SelectResource';
 import { useUrlFiltering } from 'teleport/components/hooks';
 
-import { ResourceCard } from './ResourceCard';
+import { ResourceCard, LoadingCard } from './ResourceCard';
 import SearchPanel from './SearchPanel';
 import { FilterPanel } from './FilterPanel';
 
 const RESOURCES_MAX_WIDTH = '1800px';
 
+//fetchMore size is 20 so lets add 20 loaders
+const loadingCardArray = new Array(20).fill(undefined);
```

Comment:
Is there a constant somewhere for fetchMore size you can use?

Reply:
Yeah I can do that. Also, now that I think about it, 20 doesn't include factors equal to our column sizes so I'll switch to 24